### PR TITLE
Bug 2308101: logrotate: add logrotate functionality for csi operator

### PR DESCRIPTION
**Pull request by parth-gr**
currently client operator has started using csi operator so collecting the csi logs from a new host path
hat csi-operator supports

### CODE REVIEW

File: collection-scripts/gather_ceph_logs [hunk 1]
```diff
@@ -27,6 +27,17 @@ for ns in $namespaces; do
         oc rsync -n "${ns}" "$(oc get pods -n "${ns}" --no-headers | grep "${node//./}-debug" | awk '{print $1}')":/host/var/lib/rook/"${ns}".nfs.csi.ceph.com/log "${CSI_LOG_OUTPUT_DIR}"
     }
 
+    # collect logs from the csi pods if csi is installed using csi-operator
+    csi_operator_log_collection() {
+        dbglog "collecting csi logs from node ${node}"
+        oc rsync -n "${ns}" "$(oc get pods -n "${ns}" --no-headers | grep "${node//./}-debug" | awk '{print $1}')":/host/var/lib/cephcsi/"${ns}".rbd.csi.ceph.com-ctrlplugin "${CSI_LOG_OUTPUT_DIR}"
+        oc rsync -n "${ns}" "$(oc get pods -n "${ns}" --no-headers | grep "${node//./}-debug" | awk '{print $1}')":/host/var/lib/cephcsi/"${ns}".rbd.csi.ceph.com-nodeplugin "${CSI_LOG_OUTPUT_DIR}"
+        oc rsync -n "${ns}" "$(oc get pods -n "${ns}" --no-headers | grep "${node//./}-debug" | awk '{print $1}')":/host/var/lib/cephcsi/"${ns}".cephfs.csi.ceph.com-ctrlplugin "${CSI_LOG_OUTPUT_DIR}"
```
Comment (parth-gr) [inline collection-scripts/gather_ceph_logs:35]: @yati1998 if the required host path is not present `var/lib/cephcsi/"${ns}".cephfs.csi.ceph.com-ctrlplugin` ,  the other must gather collection wont fail, right?

Comment (yati1998) [reply collection-scripts/gather_ceph_logs:35]: no it wont. Did you test the changes?

Comment (yati1998) [reply collection-scripts/gather_ceph_logs:35]: Can we add a check below before calling the function, to make sure we collect it incase of csi-operator only?

Comment (yati1998) [reply collection-scripts/gather_ceph_logs:35]: sure, we can discuss this offline

**Review by yati1998 — APPROVED** [review]

**Comment by black-dragon74** [comment]
@yati1998 Where is this function `csi_operator_log_collection` called?

**Comment by parth-gr** [comment]
> @yati1998 Where is this function `csi_operator_log_collection` called?
> 
> The changes only define the function but never call it.

Ohh , will add a commit, Thanks for catching

**Comment by yati1998** [comment]
> @yati1998 Where is this function `csi_operator_log_collection` called?
> 
> The changes only define the function but never call it.

it is called in earlier PR made by him right?

**Comment by yati1998** [comment]
okay, it's just csi_log_collection, okay my bad!!

**Pull request merged**